htdocs/hrm/lib/hrm.lib.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/* Copyright (C) 2020 jean-pascal BOUDET <jean-pascal.boudet@atm-consulting.fr>
+ * Copyright (C) 2021 Gauthier VERDOL <gauthier.verdol@atm-consulting.fr>
+ * Copyright (C) 2021 Greg Rastklan <greg.rastklan@atm-consulting.fr>
+ * Copyright (C) 2021 Jean-Pascal BOUDET <jean-pascal.boudet@atm-consulting.fr>
+ * Copyright (C) 2021 Grégory BLEMAND <gregory.blemand@atm-consulting.fr>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ * \file    hrm/lib/hrm.lib.php
+ * \ingroup hr
+ * \brief   Library files with common functions for Workstation
+ */
+
+/**
+ * Prepare admin pages header
+ *
+ * @return array
+ */
+function hrmAdminPrepareHead()
```

Comment:
We are loosing the tab that link to the existing page to setup the "department" of a company:
"admin_establishment.php" ?

Reply:
Hi @eldy, I will have a look at this on Monday. Have a nice week-end.

Reply:
Hi @eldy, i just restore link to hrm establihsment tab